salt/modules/mongodb.py

```diff
@@ -223,7 +253,7 @@ def user_exists(name, user=None, password=None, host=None, port=None,
     return False
 
 
-def user_create(name, passwd, user=None, password=None, host=None, port=None,
+def user_create(name, passwd, roles=None, user=None, password=None, host=None, port=None,
```

Comment:
Can you move the new `roles` kwarg to the end of the list of positional arguments? We don't want to cause an API change in case people are positional calls somewhere when calling out to this module.